### PR TITLE
Client is now movable during the refresh

### PR DESCRIPTION
**Pull request by Postremus**
solves one part of #839

### CODE REVIEW

**Comment by pjf** [comment]
You are awesome. :)

Tomorrow is sprint day for me, and I'm looking forward to *merging all the things* tomorrow (if I'm not beaten to it first, which is even better!).

File: GUI/Main.cs [hunk 1]
```diff
@@ -366,6 +366,13 @@ protected override void OnLoad(EventArgs e)
 
         protected override void OnFormClosing(FormClosingEventArgs e)
         {
+            //Only close the window, when the user has access to the "Exit" of the menu
+            if (!menuStrip1.Enabled)
```
Comment (pjf) [inline GUI/Main.cs]: Just checking the logic here: if the menu strip *isn't* enabled, then closing the window is treated as a cancel instead?

Comment (Postremus) [inline GUI/Main.cs]: if the menustrip isn't enabled (e.g. the user isn't allowed to click on the menustrip), we disallow closing of the form.

**Comment by pjf** [comment]
Notes for me:

- The client wasn't able to hit 'x' to close the form before this change, so it's *not* a user-visible change that they can now.
- Testing by @plague006 confirms this works as intended under Windows.
- Testing by me confirms this doesn't change behaviour under Linux (which always allows moving windows).

Merging with thanks. :)

**Pull request merged**